.rspec

```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```

Comment:
Agreed, this is just what RSpec generates by default. I'll remove color and remove `require 'spec_helper'` from existing specs.